cloud_formation/configs/dynamolambda.py

```diff
@@ -96,7 +96,8 @@ def create_config(session, domain):
                           "index.handler"),
                       timeout=120,
                       memory=128,
-                      runtime="nodejs6.10")
+                      runtime="nodejs6.10",
+                      reserved_executions=1)
```

Comment:
is this nodejs lambda the 3rd party dynamodb autoscaler? The new integrated dynamodb autoscaling capability works pretty well and may simplify deployment. It's worth considering switching to it.